csharp/examples/FluentBuilderExample/FluentBuilderExample.csproj

```diff
@@ -0,0 +1,15 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  
+  <Import Project="../../build/Common.props" />
+
+  <PropertyGroup>
+    <LangVersion>7.2</LangVersion>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>netcoreapp2.0</TargetFramework>
```

Comment:
FYI - `netcoreapp2.0` is no longer supported. This should be `netcoreapp2.1`.

[https://devblogs.microsoft.com/dotnet/net-core-2-0-will-reach-end-of-life-on-september-1-2018/](https://devblogs.microsoft.com/dotnet/net-core-2-0-will-reach-end-of-life-on-september-1-2018/)